Resources/Prototypes/Recipes/Lathes/security.yml

```
@@ -12,6 +12,13 @@
   materials:
     Plastic: 200
 
+- type: latheRecipe
+  id: BolaEnergy
+  result: BolaEnergy
+  completetime: 2
+  materials:
+    Plastic: 300
```

Since it an energy bola, it should include plasma in it's recipe. Something around 1.5 plasma sheets (150 Plasma) per ebola should be fine.
```suggestion
    Plastic: 300
    Plasma: 150
```